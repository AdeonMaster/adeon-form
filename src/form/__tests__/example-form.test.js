import React from 'react';
import { render } from '@testing-library/react';

import { Form, Field } from '..';

describe('Form', () => {
  it('should render properly', () => {
    const { container } = render(
      <Form>
        <Field name="email" type="text" defaultValue="example@gmail.com" />
      </Form>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
