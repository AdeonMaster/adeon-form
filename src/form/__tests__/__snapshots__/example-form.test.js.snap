// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render properly 1`] = `
<form>
  <input
    name="email"
    type="text"
    value="example@gmail.com"
  />
</form>
`;
